Protocol.md

```diff
@@ -139,6 +143,7 @@ The sending node SHOULD:
   - set `refund_locktime` sufficiently long after the latest possible release of oracle signatures added to all other delays to closing the contract.
   - set `payout_spk` to a previously unused script public key.
   - set `change_spk` to a previously unused script public key.
+  - set `refund_locktime` to a value not too big that both party can be expected to be refunded in a reasonable time-frame (recommended not more than latest contract maturity + 86400 * 14 meaning 2 weeks after latest contract maturity).
```

Comment:
Perhaps it'd be good to have a recommendation for minimum as well (i.e. recommended no less than 2 days after the latest contract maturity) to give the oracle time to attest in case there are any infrastructure issues

Reply:
Isn't that cover [here](https://github.com/discreetlogcontracts/dlcspecs/pull/193/files#diff-e0f5b925f91a1c09c6daf26f9a7d28816cb9ff9f08863faca719b7ee0a1cc065R137)?

Reply:
Yep you're right, looks good 👍